test/issues.jl

```diff
@@ -49,3 +49,24 @@ end
         @test_throws NoCredentials Lambda.list_functions(;aws_config=AWSConfig(creds=nothing))
     end
 end
+
+@testset "issue 324" begin
+    body = "Hello World!"
+    file_name = "streaming.bin"
+
+    try
+        S3.put_object(bucket_name, file_name, Dict("body" => body))
```

Comment:
Ah looks like when I made this I created and deleted the bucket after every test.
Probably to ensure that no artifacts get stuck and left over.

You'll just need to preface this line with:

```suggestion
        S3.create_bucket(bucket_name)
        S3.put_object(bucket_name, file_name, Dict("body" => body))
```

Reply:
done